src/migrations/migration_runner.js

```diff
@@ -76,10 +76,16 @@ export default class MigrationRunner {
     let toUpgrade = 0;
     let warning = 'The following migrations reported outdated objects:\n';
     for (const migration of this.getMigrations()) {
-      const count = await migration.count();
-      toUpgrade += count;
-      if (count > 0) {
-        warning += `${count} objects - "${migration.constructor.description}"\n`;
+      try {
+        const count = await migration.count();
+        toUpgrade += count;
+        if (count > 0) {
+          warning += `${count} objects - "${migration.constructor.description}"\n`;
+        }
+      } catch (e) {
+        this._logger.error(`Error during migration.count: ${migration.constructor.description}`);
+        console.error(e);
```

Comment:
do we need this console?

Reply:
This is the LOC responsible for the stack trace. I'm not sure whether `this._logger.error` would also parse the error and print stack trace. I can try.

Reply:
if it is needed than 
just add a comment that it is needed

Reply:
Looks beautiful enough with the `_logger`:
![image](https://user-images.githubusercontent.com/18154526/81928522-4bc90e00-95dd-11ea-9b30-78baa877a565.png)
Pushed.

Reply:
As you can see, the error message is also being logged in the third last line, that is being [caught here](https://github.com/sirensolutions/kibi-internal/blob/14632fd19948df318acd6a27c7bd5859f6145163/src/cli/kibi/upgrade.js#L162).
Alternatively, we may also ```process.stderr.write(`${error}\n`);``` -> `logger.error(error);` over there and remove the logging of error object from here.